src/components/profile/UserProfilePage/index.tsx

```diff
@@ -106,16 +103,18 @@ export const UserProfilePage = ({
           {isSignedInUser ? 'You need ' : `${handleUser.firstName} needs `}
           {100 - (handleUser.points % 100)} more points to level up to
```

Comment:
Should we hide this when you're at the last rank?

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/14285d20-c06f-4030-9a93-9b99468339e0)

Reply:
I'll make it say Level 1009 instead 👍